src/ring_theory/dedekind_domain/ideal.lean

```diff
@@ -905,6 +905,8 @@ lemma ideal.count_normalized_factors_eq {p x : ideal R} (hp0 : p ≠ ⊥) [hp :
 count_normalized_factors_eq ((ideal.prime_iff_is_prime hp0).mpr hp).irreducible
   (by { haveI : unique (ideal R)ˣ := ideal.unique_units, apply normalize_eq })
   (by convert ideal.dvd_iff_le.mpr hle) (by convert mt ideal.le_of_dvd hlt)
+/- Warning: even though a pure term-mode proof typechecks (the `by convert` can simply be
+  removed), it's slower and has caused timeout. -/
```

Comment:
Yes this is just a multiline pure comment.

It's a single timeout (or possibly zero) if you compile the library once; that's what I mean and I think it's okay?

Reply:
They are all timeouts when checking this single lemma; it's just CI compiling the whole library for every branch. If there are timeouts elsewhere in the library, this PR doesn't fix them and you need to find out the root cause.

By the way, can you confirm the speedup so maintainers can merge quickly?

Reply:
I think a contributor that want to golf this doesn't need to know for how long this timeout has bugged people in mathlib history, only that changing it will cause timeout with some probability (they should already feel the slowdown after they attempt the golf).

Reply:
I can confirm this proof is slightly faster than the old one.

Reply:
```suggestion
  removed), it's slower and has caused a timeout. -/
```

Reply:
```suggestion
  removed), it's slower to the point of a possible timeout. -/
```